create_project_via_eos.json

```diff
@@ -0,0 +1,112 @@
+{
+  "operationName": "CreateProjectMutation",
+  "variables": {
+    "input": {
+      "teamId": "<your-team-id>",
+      "externalObjectStorageId": "<your-eos-id>",
+      "name": "Demo via API",
+      "documents": [
+        {
+          "document": {
+            "name": "path/to/file/in/eos/file-1.mp3",
```

Comment:
minor, `name` here doesn't have to match objectKey exactly, so let's make them different

ie, just `file-1.mp3` & `file-1.json`

and also make sure to update `documentAssignments` as well? :thinking: